InFlightApp/Views/EntertainmentMainPage.xaml

```diff
@@ -0,0 +1,12 @@
+﻿<Page
+    x:Class="InFlightApp.Views.EntertainmentMainPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:InFlightApp.Views"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <TextBlock Text="Entertainment"></TextBlock>
+</Page>
```

Comment:
Entertainment page is de film pagina dan?

Reply:
Entertainment page is voor zowel films, muziek, series